package.json

```diff
@@ -17,7 +17,7 @@
     "test": "yarn type-check && yarn test:unit",
     "test:unit": "jest --ci",
     "type-check": "tsc --noEmit --skipLibCheck",
-    "watch:server": "nodemon ./build/server.js",
+    "watch:server": "nodemon --inspect ./build/server.js",
```

Comment:
--inspect added for node debugging

Reply:
Might be worth exposing the debugging port in the `docker-compose.yml` file too

Reply:
https://github.com/guardian/gateway/pull/123/commits/0087f1e623c0297e7ac399b715f31060c4b661ad